polaris-service/build.gradle.kts

```diff
@@ -41,6 +41,11 @@ dependencies {
     exclude("ch.qos.reload4j", "reload4j")
     exclude("log4j", "log4j")
     exclude("org.apache.zookeeper", "zookeeper")
+    exclude("org.apache.hadoop.thirdparty", "hadoop-shaded-protobuf_3_25")
```

Comment:
It's not in the LICENSE file.

Reply:
This transitive is introduced by Hadoop update.

Reply:
The other three as well?

Reply:
No, the others dependencies are coming also from hadoop for ages. I exclude these dependencies (old jersey version) because they are GPL dependencies (not ASF compliant).

Reply:
The "old" jersey dependencies are removed from the `LICENSE.bin` file in the other PR.

Reply:
@snazy do you prefer I just do the update in this PR and cleanup transitive dependency in #495 ?

Reply:
I don't mind in which PR its done, so +1